forgit.plugin.zsh

```diff
@@ -199,12 +199,19 @@ forgit::fixup() {
         --bind=\"ctrl-y:execute-silent(echo {} |grep -Eo '[a-f0-9]+' | head -1 | tr -d '[:space:]' |${FORGIT_COPY_CMD:-pbcopy})\"
         $FORGIT_FIXUP_FZF_OPTS
     "
-    commit=$(eval "$cmd" | FZF_DEFAULT_OPTS="$opts" fzf --preview="$preview" |
+    target_commit=$(eval "$cmd" | FZF_DEFAULT_OPTS="$opts" fzf --preview="$preview" |
         grep -Eo '[a-f0-9]+' | head -1)
-    [[ -n "$commit" ]] && git commit --fixup "$commit" && \
+    if [[ -n "$target_commit" ]] && git commit --fixup "$target_commit"; then
+        # "$commit~" is invalid when the commit is the first commit, but we can use "--root" instead
+        if [[ "$(git rev-parse "$target_commit")" == "$(git rev-list --max-parents=0 HEAD)" ]]; then
+            prev_commit="--root"
+        else
+            prev_commit="$commit~"
```

Comment:
We should also rename it here: "$commit~" -> "$target_commit~" (plus in the comment on line 205)

Reply:
perfect, I'm going to fix this as a part of my checkin, should be done by the end of the day!

Reply:
@ArikDavidov Thank you for quickly catching this!